core/templates/oa_hash_map.h

```diff
@@ -358,32 +363,30 @@ class OAHashMap {
 	}
 
 	void operator=(const OAHashMap &p_other) {
-		if (capacity != 0) {
-			clear();
-		}
-
+		clear();
 		_resize_and_rehash(p_other.capacity);
 
 		for (Iterator it = p_other.iter(); it.valid; it = p_other.next_iter(it)) {
 			set(*it.key, *it.value);
 		}
 	}
 
-	OAHashMap(uint32_t p_initial_capacity = 64) {
+	OAHashMap(uint32_t p_initial_capacity = 62) {
```

Comment:
I don't think it makes sense to offset this like this, should be offset internally instead

Reply:
I had previously made a commit where I split the constructor into two as in the main `hashmap`. And there were no errors.

Reply:
What do you mean? I'm talking about having to make your code `OAHashMap(x - 2)` to get what you want, you shouldn't get `128` capacity if you're asking for `63`

Reply:
Oh, you mean that.